webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/QueryProcessor.scala

```diff
@@ -53,6 +53,12 @@ trait QueryProcessor {
    */
   def process(query:Query) : Box[Seq[NodeInfo]]
 
+  /**
+   * Only get the nodeId, to update dynamic groups
+   * @param query
+   * @return
```

Comment:
Can you remove the two param/return comment, they are useless.
Also for the comment, perhaps something like: 
```
Only get node ids corresponding to that request, with minimal consistency check. This method is usefull to maximize performance (low memory, high throughout) for ex for dynamic groups. 
```